lib/iris/tests/unit/lazy_data/test_array_masked_to_nans.py

```diff
@@ -33,7 +33,7 @@
 class Test(tests.IrisTest):
     def _common_checks(self, result):
         self.assertIsInstance(result, np.ndarray)
-        self.assertFalse(isinstance(result, ma.MaskedArray))
+        self.assertFalse(ma.isMaskedArray(result))
         self.assertFalse(ma.is_masked(result))
```

Comment:
?? what does `self.assertFalse(ma.isMaskedArray(result))` mean to you.
```
isMaskedArray(x)
    Test whether input is an instance of MaskedArray.
    
    This function returns True if `x` is an instance of MaskedArray
    and returns False otherwise.  Any object is accepted as input.

```
If the type is not a masked array, it cannot be masked, so why do you want to check this ?